pkg/mcs/scheduling/server/server.go

```diff
@@ -413,7 +414,8 @@ func (s *Server) startServer() (err error) {
 	// different service modes provided by the same pd-server binary
 	bs.ServerInfoGauge.WithLabelValues(versioninfo.PDReleaseVersion, versioninfo.PDGitHash).Set(float64(time.Now().Unix()))
 	bs.ServerMaxProcsGauge.Set(float64(runtime.GOMAXPROCS(0)))
-	deployPath, err := os.Executable()
+	execPath, err := os.Executable()
```

Comment:
tso and resource manager also need to update

Reply:
get it. only tso and scheduling have it

Reply:
Do we also need to pick it to release-8.0?

Reply:
added